### ISSUE TITLE
We released a similar implementation of this project - `@strv/nestjs-dataloader`

### ISSUE THREAD

**Issue by robertrossmann** [krislefeber/nestjs-dataloader#64]
Hi @krislefeber,

tl;dr - we recently released an updated and improved version of your package with similar but not identical API: https://github.com/strvcom/nestjs-dataloader ([npm](https://www.npmjs.com/package/@strv/nestjs-dataloader)).

---

At [STRV](https://www.strv.com), we are developing GraphQL backends as part of our business and we need a good Dataloader integration with Nest.js. We initially wanted to use your package as it provided exactly what we wanted and needed but seeing as this repository has not been receiving updates for some time and support for latest Nest.js version was missing, we decided to re-implement a similar functionality from scratch while taking a lot of inspiration from your work. As we are building a business on top of Nest.js and GraphQL we expect our package to receive regular updates in the foreseeable future.

If you do not plan on working on this project anymore (understandable!) but like our version of it, please consider taking steps that would direct your community towards our package as this repo still seems to receive a lot of traffic. This might include:

- updating your README with a note to check out our work
- archiving the repo and/or the npm package and informing your dependents of our alternative
- other

Again, thank you and your community for your work on this package, it was a great inspiration for us! ❤️

**Comment by kasir-barati** [comment]
TBH I was planning to use it but then I saw your license change and then I recalled with what happened in [Terraform](https://www.reddit.com/r/devops/comments/195kya8/working_with_new_terraform_licenses/).

**Comment by robertrossmann** [comment]
@kasir-barati there was no license change, the repo was still private when we changed the license to BSD-3-Clause.

**Comment by kasir-barati** [comment]
OK, so basically your project is not a fork of this one. My bad. I though it was a Fork and then license change 🤦. Thanks for the clarification.
